packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -71,7 +86,7 @@ abstract class SemanticsEvent {
 class AnnounceSemanticsEvent extends SemanticsEvent {
 
   /// Constructs an event that triggers an announcement by the platform.
-  const AnnounceSemanticsEvent(this.message, this.textDirection)
+  const AnnounceSemanticsEvent(this.message, this.textDirection, this.assertiveness)
```

Comment:
Can adding a third parameter like this be a breaking change? Existing code may be calling this constructor as `AnnounceSemanticsEvent('foo', TextDirection.ltr)`. A non-breaking way would be to make it a named parameter like the change in the `announce` function.

Reply:
+1 to making this a named parameter with default value `polite`.